pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "paperless-ngx"
-version = "0.1.0"
+version = "2.14.7"
```

Comment:
Something we should do for releases going forward?

Reply:
Not yet, though maybe in the future